pkg/cmd/pipeline/describe_test.go

```diff
@@ -37,10 +37,10 @@ func TestPipelineDescribe_Empty(t *testing.T) {
 
 	pipeline := Command(p)
 	output, err := test.ExecuteCommand(pipeline, "desc", "bar")
-	if err != nil {
+	if err == nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
-	expected := "Error : pipelines.tekton.dev \"bar\" not found\n"
+	expected := "Error: pipelines.tekton.dev \"bar\" not found\n"
 	if d := cmp.Diff(expected, output); d != "" {
```

Comment:
output should be `Error.error` isnt it ?

Reply:
Yeh, make sense.